crates/holochain_conductor_api/Cargo.toml

```diff
@@ -21,6 +21,9 @@ holochain_state_types = { version = "^0.5.0-dev.6", path = "../holochain_state_t
 holochain_serialized_bytes = "=0.0.55"
 holochain_types = { version = "^0.5.0-dev.12", path = "../holochain_types" }
 holochain_zome_types = { version = "^0.5.0-dev.10", path = "../holochain_zome_types" }
+holochain_util = { version = "^0.5.0-dev.0", path = "../holochain_util", features = [
```

Comment:
Do you need any of the other features? Otherwise I'd use `default-features = false` here to avoid bringing in anything extra